pkg/sync/v14/data/SecurityContextConstraints.security.openshift.io/privileged.yaml

```diff
@@ -41,6 +41,7 @@ users:
 - system:serviceaccount:openshift-infra:build-controller
 - system:serviceaccount:openshift-etcd:etcd-backup
 - system:serviceaccount:openshift-azure-logging:log-analytics-agent
+- system:serviceaccount:openshift-azure-logging:omsagent
```

Comment:
@Makdaam can you please have a look at this? I can't find a way to replace `log-analytics` with `omsagent` (not even edit this as cluster-admin). Can we add `omsagent` for now and remove `log-analytics` later or is there a better way to do it?

Reply:
If you'd like to replace it for testing on your dev cluster you can
`oc annotate scc privileged openshift.io/reconcile-protect=true`
as a cluster-admin.
Otherwise the sync pod will reset it to what you have compiled in the azure image pkg/sync/vX/data.

If this is about replacing the accounts on prod, then I'd do it like you described: add a new account, migrate, remove old account. Or if it can be synced with our release cycle just flip the switch at once.

Reply:
@Makdaam The problem is not the sync-pod. (in my setup it is scaled down to 0 and I only run `make sync-run`) The problem that if I replace `log-analytics` with `omsagent` in SCC in pkg/sync/vX/data and run the sync-pod I get: 

```
INFO[2019-12-13T09:46:37+11:00] Update SecurityContextConstraints.security.openshift.io/privileged 
INFO[2019-12-13T09:46:37+11:00]   unstructured.Unstructured{
        Object: map[string]interface{}{
                ... // 20 identical entries
                "seccompProfiles":    []interface{}{string("*")},
                "supplementalGroups": map[string]interface{}{"type": string("RunAsAny")},
                "users": []interface{}{
                        string("system:admin"),
                        string("system:serviceaccount:openshift-infra:build-controller"),
                        string("system:serviceaccount:openshift-etcd:etcd-backup"),
-                       string("system:serviceaccount:openshift-azure-logging:log-analytics-agent"),
                        string("system:serviceaccount:openshift-azure-logging:omsagent"),
-                       string("system:serviceaccount:openshift-azure-logging:newsagent"),
                        string("system:serviceaccount:kube-system:sync"),
                },
                "volumes": []interface{}{string("*")},
        },
  } 
INFO[2019-12-13T09:46:37+11:00] sync error: admission webhook "sccs.aro-admission-controller.aro.openshift.io" denied the request: users: Required value: must include user system:serviceaccount:openshift-azure-logging:log-analytics-agent 
```

So if there is no issue with adding the "omsagent" first and then in future remove "log-analytics" I think it would be the safest approach.